code-challenges.js

```diff
@@ -9,67 +9,207 @@
 // --------------------1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Fahrenheit.
 // Use the test variables provided below. Expected outcome: "85 is below boiling point" "350 is above boiling point" "212 is at boiling point"
 
+//Pseudocode
+//create a function
+//argument will be a number in a variable
+//inside function use ifelse clauses to determine if the number is >/</= to 212
+//depending on the outcome output string says whether it is >/</= to 212
+//console.log(function(number)); to test result
+
 var temp1 = 85
 var temp2 = 350
 var temp3 = 212
 
-
-
+const boilPoint = (temp) => {
+  if(temp < 212){
+    return `${temp} is below boiling point`
+  }else if(temp > 212){
+    return `${temp} is above boiling point`
+  }else if(temp = 212){
```

Comment:
`temp = 212` is not the same as `temp === 212`. The first one is an assignment of the value 212 to temp whereas the latter is an equality comparison